app/public/templates/scene.html

```diff
@@ -190,22 +195,26 @@
           {% end %}
           inspector.openInspector();
       });
-      async function takeSceneScreenshot(){
-          const screenshot_canvas = scene.components.screenshot.getCanvas('perspective');
-          await uploadScreenshotToS3(screenshot_canvas.toDataURL("image/jpeg"));
+      async function takeSceneScreenshot(s3_key=""){
+        if(initial_load_js || (!initial_load_js && s3_key !== "")){
```

Comment:
Minor thing, can we make this into `if(initial_load_js || s3_key !== ""){`?
Unless you kept it like this intentionally for readability

Reply:
Agreed. From an outside perspective, I think `initial_load_js || (!initial_load_js && s3_key !== "")` is less readable than what Dhruvin suggested here. And yeah, appears logically equivalent to me.